### ISSUE TITLE
Portugal numbers aren't being updated

### ISSUE THREAD

**Issue by gui710** [CSSEGISandData/COVID-19#583]
Portugal already has about 79 confirmed cases yet your map still says 59.
This one shows the numbers almost in real time and is using reputable sources.
https://pt-covid-19.maps.arcgis.com/apps/opsdashboard/index.html?fbclid=IwAR2aokoFjCrcnigkxk_aTZ2Z1dFhz-HABz_FVQSvbEUlm6iS1g4CXVXazQ4#/0f585619e8e241ad895b65575e24c57d

**Comment by Syderitic** [comment]
Yeah I think the maintenance that they are talking about is having some impact on this.
